src/themes/greyVest/TreePauseButton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import { observer } from 'mobx-react'
+import IconButton from './IconButton'
+
+let setPausedNested = (tree, path, value) =>
+  tree[`${value ? '' : 'un'}pauseNested`](path)
+
+let TreePauseButton = ({ children }) => {
```

Comment:
Remember that the idea is not to force you into `SearchEditor`. This button will work anywhere you want to toggle the paused values of one or more trees, and it would be necessary if you want to customize the controls shown at the top. It's a pretty self-contained thing, so it definitely makes sense to not just be a part of `SearchEditor` internally - and exporting it helps in a non-`SearchEditor` scenarios

Reply:
>  it would be necessary if you want to customize the controls shown at the top

> the idea is not to force you into SearchEditor

Is this something we're doing right now or is it future-proofing? In general, I try to not do the latter things unless it's extremely obvious.